sdl_android/src/main/java/com/smartdevicelink/proxy/SdlProxyBuilder.java

```diff
@@ -39,7 +40,9 @@
 	private boolean preRegister;
 	private String sAppResumeHash;
 	private BaseTransportConfig mTransport;
-	private List<Class<? extends SdlSecurityBase>> sdlSecList;	
+	private List<Class<? extends SdlSecurityBase>> sdlSecList;
+	private TemplateColorScheme dayColorScheme = null;
```

Comment:
This can be one line `private TemplateColorScheme dayColorScheme  = null, nightColorScheme = null;`